setup.py

```diff
@@ -24,7 +24,7 @@ def get_path(fname):
 
 
 install_requirements = [
-    "Django>=1.11,<2.1",
+    "Django>=1.11,<2.1.1",
```

Comment:
This comes with the same issues that I mentioned in https://github.com/django-wiki/django-nyt/pull/82

We cannot allow Django 2.1 without adding it to the test matrix, fixing whatever issues come along.